service/routes.py

```diff
@@ -21,7 +21,7 @@
 and Delete Pets from the inventory of pets in the PetShop
 """
 
-from flask import jsonify, request, url_for, abort
+from flask import jsonify, request, abort
 from flask import current_app as app  # Import Flask application
 from service.models import YourResourceModel
```

Comment:
Does this need to be changed? "YourResourceModel"

Reply:
this is fine for now, we will change this once `PaymentMethod` is merged